### ISSUE TITLE
Rails 4.2 support

### ISSUE THREAD

**Issue by onyoo** [grempe/tierion#2]
Hi,

Curious if there is some way to use this gem with Rails 4.2. I tried adding it to my gemfile, but can't get the dependencies to match on activesupport.

```
In Gemfile:
    rails (= 4.2.6) was resolved to 4.2.6, which depends on
      actionview (= 4.2.6) was resolved to 4.2.6, which depends on
        activesupport (= 4.2.6)

....

tierion was resolved to 0.1.0, which depends on
      activesupport (~> 5.0)
```

We're looking into using Rails 5 in a future version, but would love to use this gem instead of writing custom code for our current build.

**Comment by onyoo** [comment]
I've forked and changed the dependency to "~>4.0". I'll keep testing it and make a pull request if no major issues arise in the next week or so.

**Comment by grempe** [comment]
I think you should be OK.  My use of activesupport is pretty limited. I only pull in the following modules:

https://github.com/grempe/tierion/blob/master/lib/tierion.rb#L2

If all seems well (I know, this lib needs a test suite) please submit a pull request for `~> 4` so it'll work with 4.x or 5.x

Its also possible that `active_support/core_ext/integer/time` is no longer needed.  If I remember correctly, I think I was using the `.months` or `.years` helper at some point and I had to pull that in since (inexplicably) those two are in integer/time but the rest (e.g. .minutes) are in `numeric`. Strange.

The time require may no longer even be needed since I no longer use these. It may not hurt to leave it though since its a strange split in the activesupport code.

https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/integer/time.rb

**Comment by grempe** [comment]
Fix merged from PR #3

**Comment by grempe** [comment]
Version 1.3.1 pushed to github and rubygems.

https://rubygems.org/gems/tierion/versions/1.3.1
